### PR TITLE
Batched tile tests

### PR DESCRIPTION
**Pull request by lilleyse**
Batched tile tests. Due to the changes in #3259 there are no cases where the model's promise is rejected for Batched tiles, so I'm missing that code coverage.

### CODE REVIEW

**Comment by pjcozzi** [comment]
Part of #3177.

File: Specs/Scene/Batched3DModel3DTileContentProviderSpec.js [hunk 1]
```diff
@@ -0,0 +1,214 @@
...
+
+    afterAll(function() {
+        scene.destroyForSpecs();
+    });
+
+    afterEach(function() {
+        scene.primitives.removeAll();
+    });
+
+    function verifyRender(tileset) {
```
Comment (pjcozzi) [inline Specs/Scene/Batched3DModel3DTileContentProviderSpec.js]: Are we ready to make a helper class to reuse this function and friends with Instanced3DModel3DTileContentProviderSpec.js and future tests?

**Comment by pjcozzi** [comment]
This looks really good.  Just that one question.

File: Specs/Scene/Batched3DModel3DTileContentProviderSpec.js (hunk 1 above)
Comment (lilleyse) [inline Specs/Scene/Batched3DModel3DTileContentProviderSpec.js]: I think I'm going to do that in a different pull request.

Comment (pjcozzi) [inline Specs/Scene/Batched3DModel3DTileContentProviderSpec.js]: OK with me.
